test/services/supplementary-billing/fetch-charge-versions.service.test.js

```diff
@@ -154,7 +154,7 @@ describe('Fetch Charge Versions service', () => {
       it('returns no applicable charge versions', async () => {
         const result = await FetchChargeVersionsService.go(regionId, billingPeriod)
 
-        expect(result.length).to.equal(0)
+        expect(result).to.have.length(0)
```

Comment:
Just a thought, do you think it would maybe read better to use `.to.be.empty()` in place of `.to.have.length(0)` for this and the other tests like it?

Reply:
Yes I think it would read better like that. I'll make the changes.

Didn't think you'd be able to find anything on this one :grinning:

Reply:
I've implemented your suggestion